apps/backend/src/main.ts

```diff
@@ -9,13 +9,13 @@ const UsersRouter = require('./app/users/user.controller').UsersRouter;
 const AuthRouter = require('./app/auth/auth.routes').AuthRouter;
 const InstitutionsRouter = require('./app/institution/institution.controller').InstitutionRouter;
 const EventsRouter = require('./app/events/event.controller').EventsRouter;
-const MailRouter = require('./app/services/mail/mail.routes').MailRouter;
+const OcurrenceEventRoutes = require('./app/ocurrence-event/ocurrence-event.controller').OcurrenceEventRoutes;
```

Comment:
```suggestion
const OcurrenceEventRoutes = require('./app/ocurrence-event/ocurrence-event.controller').OcurrenceEventRoutes;
```
Por error de tipeo reemplazar por:
const OcurrenceEventRoutes = require('./app/occurrence-events/occurrence-events.controller').OcurrenceEventRoutes;